### DISCUSSION TITLE
Add the 4-point drag icon to Entries fields

### DISCUSSION THREAD

**Discussion by darylknight** [craftcms/cms#9013] [Ideas]
### Description

A client has just pointed out that they didn't realise they could drag entries in an Entries field to re-order them, because normally elsewhere in the control panel there's an icon indicating you can do that, like this:

![image](https://user-images.githubusercontent.com/1250017/112619709-4e68d680-8e30-11eb-803b-06218284d9e2.png)

On an entries field, you can re-order them by dragging, but there's no icon:

![image](https://user-images.githubusercontent.com/1250017/112619645-3ee98d80-8e30-11eb-9ce9-2cdcbb3066fa.png)

Wondering if it would be more consistent to include this.

**Comment by wedot** [comment]
+1 🎉 please add the same for the category field

**Comment by brandonkelly** [comment]
Categories fields aren’t sortable in the first place.

**Comment by wedot** [comment]
@brandonkelly Craft CMS should have this option.

**Comment by brandonkelly** [comment]
@wedot It’s already on the list for 4.0 – see #7332.
